config/initializers/sentry.rb

```diff
@@ -0,0 +1,26 @@
+Sentry.init do |config|
+  # Sentry config here is mostly copied from apply-for-teacher-training
+  # https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/config/initializers/sentry.rb
+  config.environment = ActiveSupport::EnvironmentInquirer.new(ENV.fetch("HOSTING_ENV", "development"))
+  config.release = ENV["SHA"]
+  config.dsn = ENV.fetch("SENTRY_DSN", "")
+
+  config.inspect_exception_causes_for_exclusion = true
+
+  config.excluded_exceptions += [
+    # The following exceptions are user-errors that aren't actionable, and can be
+    # safely ignored.
+    "ActionController::BadRequest",
+    "ActionController::UnknownFormat",
+    "ActionController::UnknownHttpMethod",
+    "ActionDispatch::Http::Parameters::ParseError",
+    "Mime::Type::InvalidMimeType",
```

Comment:
I think we can still get some value from some of these errors. If we have a lot of `BadRequest` it still might tell us something, maybe our form is not right or something else 🤷🏻